### ISSUE TITLE
TODO List

### ISSUE THREAD

**Issue by hexylena** [galaxy-genome-annotation/dockerized-gmod-deployment#2]
- [x] @abretaud is preparing some time delays
- [x] sockjs fixed
- [x] any other delays or does it come up OK now?
- [ ] training materials

**Comment by abretaud** [comment]
I tried delaying apollo startup but I still get some sql errors at startup: https://gist.github.com/abretaud/023846c9fa8085b3dab120f5f94b93f1
It looks like it's trying to make some changes that are already done, except maybe the last error
@nathandunn do you have some clue on this?

There are also some SQL errors when drupal gets installed too, but it seems to happen when it checks wether it is already installed or not, so probably harmless

**Comment by hexylena** [comment]
yeah, drupal's errors are harmless. They're quite good about safe migrations it seems.

**Comment by abretaud** [comment]
Probably not high priority, but looking at the tripal log, there are messages about a missing uploadprogress.so: the module installation [here](https://github.com/erasche/docker-tripal/blob/master/Dockerfile#L24) fails because it is not compatible with php 7. There is a modified version [there](https://bugs.php.net/bug.php?id=69242) but not yet available on PECL...

**Comment by nathandunn** [comment]
There are some random ones from Apollo when it sets up Chado, but those should be harmless.  

Nathan
